viz/charts/linkerd-viz/templates/namespace.yaml

```diff
@@ -10,5 +10,11 @@ metadata:
   labels:
     {{.Values.extensionAnnotation}}: linkerd-viz
   annotations:
+    {{- if .Values.prometheusUrl }}
+    viz.linkerd.io/external-prometheus: {{.Values.prometheusUrl}}
+    {{- end }}
+    {{- if .Values.grafanaUrl }}
```

Comment:
This logic i.e assuming `linkerd-grafana` is presently based on the absence of this annotation does not really work with grafana because both being absent case is allowed here. Should we make disabled an explicit annotation then?